internal/services/cosmos/cosmosdb_postgresql_cluster_resource.go

```diff
@@ -636,3 +659,19 @@ func flattenMaintenanceWindow(input *clusters.MaintenanceWindow) []MaintenanceWi
 		},
 	}
 }
+
+func formatServerNames(input *[]clusters.ServerNameItem) []ServerNameItem {
+	if input == nil {
+		return nil
+	}
+
+	var output []ServerNameItem
```

Comment:
We should be setting an empty value into state instead of nil here 
```suggestion
	var output []ServerNameItem
	if input == nil {
		return output
	}

```